samples/javascript_nodejs/14.nlp-with-dispatch/index.js

```diff
@@ -8,46 +8,41 @@ const path = require('path');
 const restify = require('restify');
 
 // Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
-const { BotFrameworkAdapter, MemoryStorage, ConversationState, UserState } = require('botbuilder');
-// Import required bot configuration.
-const { BotConfiguration } = require('botframework-config');
+const { BotFrameworkAdapter } = require('botbuilder');
+const { LuisService } = require('botframework-config');
```

Comment:
Since we've removed the .bot file, we should also remove the dependency on `botframework-config`. I realize this differs from the C# sample, but the `LuisService` is just a data model. 

We don't have the `botframework-config` dependency in `botbuilder-ai` that `Microsoft.Bot.Builder.Ai.LUIS` has on `Microsoft.Bot.Configuration`, so let's get rid of the dependency.

Reply:
aye-aye!

removing